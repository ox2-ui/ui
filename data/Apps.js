export const items = [
  {
    _id: '1nNKyP6h6pMMjAyTMK',
    title: 'Smartup Visuals',
    idTag: 'smartup',
    logo: 'http://imgur.com/BC6B8Bc.png',
  },
  {
    _id: '2nNKyP6h6pMMjAyTMK',
    title: 'ACI Events',
    idTag: 'aci',
    logo: 'http://imgur.com/N2LoYmv.png',
  },
  {
    _id: '3nNKyP6h6pMMjAyTMK',
    title: 'Aircraft Commerce',
    idTag: 'aircraft',
    logo: 'http://imgur.com/abQ9EaG.png',
  },
  {
    _id: '4nNKyP6h6pMMjAyTMK',
    title: 'ICC Events',
    idTag: 'icc',
    logo: 'http://imgur.com/UmS4Irj.png',
  },
  {
    _id: '5nNKyP6h6pMMjAyTMK',
    title: 'Smartup Visuals',
    idTag: 'smartup',
    logo: 'http://imgur.com/BC6B8Bc.png',
  },
  {
    _id: '6nNKyP6h6pMMjAyTMK',
    title: 'ACI Events',
    idTag: 'aci',
    logo: 'http://imgur.com/N2LoYmv.png',
  },
  {
    _id: '7nNKyP6h6pMMjAyTMK',
    title: 'Aircraft Commerce',
    idTag: 'aircraft',
    logo: 'http://imgur.com/abQ9EaG.png',
  },
  {
    _id: '8nNKyP6h6pMMjAyTMK',
    title: 'ICC Events',
    idTag: 'icc',
    logo: 'http://imgur.com/UmS4Irj.png',
  },
  {
    _id: '9nNKyP6h6pMMjAyTMK',
    title: 'Smartup Visuals',
    idTag: 'smartup',
    logo: 'http://imgur.com/BC6B8Bc.png',
  },
  {
    _id: '10nNKyP6h6pMMjAyTMK',
    title: 'ACI Events',
    idTag: 'aci',
    logo: 'http://imgur.com/N2LoYmv.png',
  },
  {
    _id: '11nNKyP6h6pMMjAyTMK',
    title: 'Aircraft Commerce',
    idTag: 'aircraft',
    logo: 'http://imgur.com/abQ9EaG.png',
  },
  {
    _id: '12nNKyP6h6pMMjAyTMK',
    title: 'ICC Events',
    idTag: 'icc',
    logo: 'http://imgur.com/UmS4Irj.png',
  },
];
