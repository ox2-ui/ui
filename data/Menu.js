export const items = [
  // System
  {
    value: '/system/status',
    iconGroup: 'linear',
    icon: 'speed-fast',
    title: 'Status',
    module: 'status',
    moduleGroup: 'system',
  },
  {
    value: '/system/apps',
    iconGroup: 'linear',
    icon: 'laptop-phone',
    title: 'Apps',
    module: 'apps',
    moduleGroup: 'system',
  },
  {
    value: '/system/servers',
    iconGroup: 'linear',
    icon: 'server',
    title: 'Servers',
    module: 'servers',
    moduleGroup: 'system',
  },
  {
    value: '/system/versions',
    iconGroup: 'linear',
    icon: 'site-map',
    title: 'Versions',
    module: 'versions',
    moduleGroup: 'system',
  },
  {
    value: '/system/roles',
    iconGroup: 'linear',
    icon: 'user-lock',
    title: 'Roles',
    module: 'roles',
    moduleGroup: 'system',
  },
  // Business
  {
    value: '/business/status',
    iconGroup: 'linear',
    icon: 'chart-growth',
    title: 'Business',
    module: 'status',
    moduleGroup: 'business',
  },
  {
    value: '/business/reports',
    iconGroup: 'linear',
    icon: 'file-charts',
    title: 'Reports',
    module: 'reports',
    moduleGroup: 'business',
  },
  {
    value: '/business/clients',
    iconGroup: 'linear',
    icon: 'briefcase',
    title: 'Clients',
    module: 'clients',
    moduleGroup: 'business',
  },
  {
    value: '/business/accounting',
    iconGroup: 'linear',
    icon: 'cashier',
    title: 'Accounting',
    module: 'accounting',
    moduleGroup: 'business',
  },
  {
    value: '/business/partners',
    iconGroup: 'linear',
    icon: 'feather2',
    title: 'Partners',
    module: 'partners',
    moduleGroup: 'business',
  },
  // Templates
  {
    value: '/templates/themes',
    iconGroup: 'linear',
    icon: 'palette',
    title: 'Themes',
    module: 'themes',
    moduleGroup: 'templates',
  },
  {
    value: '/templates/screens',
    iconGroup: 'linear',
    icon: 'screen',
    title: 'Screens',
    module: 'screens',
    moduleGroup: 'templates',
  },
  {
    value: '/templates/badges',
    iconGroup: 'moon',
    icon: 'qrcode',
    title: 'Badges',
    module: 'badges',
    moduleGroup: 'templates',
  },
  // App
  {
    value: '/app/settings',
    iconGroup: 'linear',
    icon: 'cog',
    title: 'Settings',
    module: 'settings',
    moduleGroup: 'app',
  },
  {
    value: '/app/assets',
    iconGroup: 'linear',
    icon: 'file-charts',
    title: 'Assets',
    module: 'assets',
    moduleGroup: 'app',
  },
  {
    value: '/app/themes',
    iconGroup: 'linear',
    icon: 'palette',
    title: 'Themes',
    module: 'themes',
    moduleGroup: 'app',
  },
  {
    value: '/app/screens',
    iconGroup: 'linear',
    icon: 'screen',
    title: 'Screens',
    module: 'screens',
    moduleGroup: 'app',
  },
  {
    value: '/app/badges',
    iconGroup: 'moon',
    icon: 'qrcode',
    title: 'Badges',
    module: 'badges',
    moduleGroup: 'app',
  },
  // Client
  {
    value: '/client/invoices',
    iconGroup: 'linear',
    icon: 'receipt',
    title: 'Invoices',
    module: 'invoices',
    moduleGroup: 'client',
  },
  {
    value: '/client/contacts',
    iconGroup: 'linear',
    icon: 'profile',
    title: 'Contacts',
    module: 'contacts',
    moduleGroup: 'client',
  },
  {
    value: '/client/branding',
    iconGroup: 'linear',
    icon: 'cube',
    title: 'Branding',
    module: 'branding',
    moduleGroup: 'client',
  },
  {
    value: '/client/apps',
    iconGroup: 'linear',
    icon: 'smartphone',
    title: 'Apps',
    module: 'apps',
    moduleGroup: 'client',
  },
];

