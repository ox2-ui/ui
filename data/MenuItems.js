export const menuItems = [
  // System
  {
    value: '/system/status',
    icon: 'speed-fast',
    title: 'Status',
    moduleGroup: 'system',
  },
  {
    value: '/system/apps',
    icon: 'laptop-phone',
    title: 'Apps',
    moduleGroup: 'system',
  },
  {
    value: '/system/servers',
    icon: 'server',
    title: 'Servers',
    moduleGroup: 'system',
  },
  {
    value: '/system/versions',
    icon: 'site-map',
    title: 'Versions',
    moduleGroup: 'system',
  },
  {
    value: '/system/roles',
    icon: 'user-lock',
    title: 'Roles',
    moduleGroup: 'system',
  },
  // Business
  {
    value: '/business/status',
    icon: 'chart-growth',
    title: 'Business',
    moduleGroup: 'business',
  },
  {
    value: '/business/reports',
    icon: 'file-charts',
    title: 'Reports',
    moduleGroup: 'business',
  },
  {
    value: '/business/clients',
    icon: 'briefcase',
    title: 'Clients',
    moduleGroup: 'business',
  },
  {
    value: '/business/accounting',
    icon: 'cashier',
    title: 'Accounting',
    moduleGroup: 'business',
  },
  {
    value: '/business/partners',
    icon: 'feather2',
    title: 'Partners',
    moduleGroup: 'business',
  },
  // Templates
  {
    value: '/templates/themes',
    icon: 'palette',
    title: 'Themes',
    moduleGroup: 'templates',
  },
  {
    value: '/templates/screens',
    icon: 'screen',
    title: 'Screens',
    moduleGroup: 'templates',
  },
  {
    value: '/templates/badges',
    icon: 'qrcode',
    title: 'Badges',
    moduleGroup: 'templates',
  },
  // App
  {
    value: '/app/settings',
    icon: 'cog',
    title: 'Settings',
    moduleGroup: 'app',
  },
  {
    value: '/app/assets',
    icon: 'file-charts',
    title: 'Assets',
    moduleGroup: 'app',
  },
  {
    value: '/app/themes',
    icon: 'palette',
    title: 'Themes',
    moduleGroup: 'app',
  },
  {
    value: '/app/screens',
    icon: 'screen',
    title: 'Screens',
    moduleGroup: 'app',
  },
  {
    value: '/app/badges',
    icon: 'qrcode',
    title: 'Badges',
    moduleGroup: 'app',
  },
  // Client
  {
    value: '/client/invoices',
    icon: 'receipt',
    title: 'Invoices',
    moduleGroup: 'client',
  },
  {
    value: '/client/contacts',
    icon: 'profile',
    title: 'Contacts',
    moduleGroup: 'client',
  },
  {
    value: '/client/branding',
    icon: 'cube',
    title: 'Branding',
    moduleGroup: 'client',
  },
  {
    value: '/client/apps',
    icon: 'smartphone',
    title: 'Apps',
    moduleGroup: 'client',
  },
];

export const iconBarItems = [
  {
    value: '@ox2/toggle',
    image: 'http://imgur.com/abQ9EaG.png',
  },
  {
    value: '@ox2/button',
    image: 'http://imgur.com/N2LoYmv.png',
  },
  {
    value: '@ox2/ui',
    image: 'http://imgur.com/BC6B8Bc.png',
  },
  {
    value: '@tapfuse/module-agenda',
    image: 'http://imgur.com/UmS4Irj.png',
  },
  {
    value: '@ox2/toggle',
    image: 'http://imgur.com/abQ9EaG.png',
  },
  {
    value: '@ox2/button',
    image: 'http://imgur.com/N2LoYmv.png',
  },
  {
    value: '@ox2/ui',
    image: 'http://imgur.com/BC6B8Bc.png',
  },
  {
    value: '@tapfuse/module-agenda',
    image: 'http://imgur.com/UmS4Irj.png',
  },
  {
    value: '@ox2/toggle',
    image: 'http://imgur.com/abQ9EaG.png',
  },
  {
    value: '@ox2/button',
    image: 'http://imgur.com/N2LoYmv.png',
  },
  {
    value: '@ox2/ui',
    image: 'http://imgur.com/BC6B8Bc.png',
  },
  {
    value: '@tapfuse/module-agenda',
    image: 'http://imgur.com/UmS4Irj.png',
  },
];

export const items = [
  {
    value: '@ox2/toggle',
    image: 'http://imgur.com/abQ9EaG.png',
  },
  {
    value: '@ox2/button',
    image: 'http://imgur.com/N2LoYmv.png',
  },
  {
    value: '@ox2/ui',
    image: 'http://imgur.com/BC6B8Bc.png',
  },
  {
    value: '@tapfuse/module-agenda',
    image: 'http://imgur.com/UmS4Irj.png',
  },
  {
    value: '@ox2/toggle',
    image: 'http://imgur.com/abQ9EaG.png',
  },
  {
    value: '@ox2/button',
    image: 'http://imgur.com/N2LoYmv.png',
  },
  {
    value: '@ox2/ui',
    image: 'http://imgur.com/BC6B8Bc.png',
  },
  {
    value: '@tapfuse/module-agenda',
    image: 'http://imgur.com/UmS4Irj.png',
  },
  {
    value: '@ox2/toggle',
    image: 'http://imgur.com/abQ9EaG.png',
  },
  {
    value: '@ox2/button',
    image: 'http://imgur.com/N2LoYmv.png',
  },
  {
    value: '@ox2/ui',
    image: 'http://imgur.com/BC6B8Bc.png',
  },
  {
    value: '@tapfuse/module-agenda',
    image: 'http://imgur.com/UmS4Irj.png',
  },
];
