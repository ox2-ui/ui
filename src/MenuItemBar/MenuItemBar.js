import React, { Component, PropTypes } from 'react';
import SelectIndicator from '../SelectIndicator';
import Scroller from '@ox2/scroller/Scroller';
import styled from 'styled-components';
import Immutable from 'immutable';
import MenuItem from '../MenuItem';

const StyledScroller = styled(Scroller)`
  padding-top: 14px;
  padding-bottom: 14px;
  position: absolute;
  top: 0;
  bottom: 0;
`;

/**
 * IconBar Component
 */

class IconBar extends Component {
  static propTypes = {
    /**
     * The css class name of the root element.
     */
    className: PropTypes.string,
    /**
     * Menu items to render
     */
    items: PropTypes.instanceOf(Immutable.List).isRequired,
    /**
     * Callback function fired when the menu item is selected
     */
    onUpdate: PropTypes.func.isRequired,
    /**
     * Override the inline-styles of the root element.
     */
    style: PropTypes.object,
    /**
     * Value to select
     */
    value: PropTypes.string.isRequired,
  };

  shouldComponentUpdate(nextProps) {
    if (this.props.items !== nextProps.value) {
      return true;
    } else if (!Immutable.is(this.props.items, nextProps.items)) {
      return true;
    } else {
      return false;
    }
  }

  render() {
    const { className, style, items, onUpdate, value } = this.props;

    return (
      <StyledScroller className={className} style={style}>
        {items.map((item, i) => {
          return (
            <div key={i} className="display:flex pos:relative padding-l:7">
              {value === item._id &&
                <SelectIndicator className="color:neutral-dim" />
              }
              <MenuItem
                value={item._id}
                title={item.title}
                image={item.logo}
                pictureWidth={35}
                pictureHeight={35}
                pictureBgColor={'white'}
                onClick={onUpdate}
              />
            </div>
          );
        })}
      </StyledScroller>
    );
  }
}

export default IconBar;
