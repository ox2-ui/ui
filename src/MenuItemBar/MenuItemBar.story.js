import React from 'react';
import { storiesOf } from '@kadira/storybook';
import MenuItemBar from './MenuItemBar';
import Immutable from 'immutable';
import { items } from '../../data/Apps';

const handler = (value) => console.log('🐳', value); // eslint-disable-line no-console

const list = Immutable.List.of(...items);

storiesOf('MenuItemBar', module)
  .add('default', () => (
    <MenuItemBar
      value="3nNKyP6h6pMMjAyTMK"
      items={list}
      onUpdate={handler}
    />
  ));
