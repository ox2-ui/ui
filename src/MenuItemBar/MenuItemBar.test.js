/* eslint-disable no-unused-vars */
import React from 'react';
import { shallow, render, mount } from 'enzyme';
import MenuItemBar from './MenuItemBar';
import Immutable from 'immutable';
import { items } from '../../data/Apps';

const handler = () => true;

const list = Immutable.List.of(...items);

it('renders correctly', () => {
  const wrapper = mount(
    <MenuItemBar
      value="3nNKyP6h6pMMjAyTMK"
      items={list}
      onUpdate={handler}
    />,
  );

  expect(wrapper).toMatchSnapshot();
});
