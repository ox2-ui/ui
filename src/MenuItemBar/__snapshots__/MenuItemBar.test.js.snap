exports[`test renders correctly 1`] = `
<IconBar
  items={
    Array [
      Object {
        "_id": "1nNKyP6h6pMMjAyTMK",
        "idTag": "smartup",
        "logo": "http://imgur.com/BC6B8Bc.png",
        "title": "Smartup Visuals",
      },
      Object {
        "_id": "2nNKyP6h6pMMjAyTMK",
        "idTag": "aci",
        "logo": "http://imgur.com/N2LoYmv.png",
        "title": "ACI Events",
      },
      Object {
        "_id": "3nNKyP6h6pMMjAyTMK",
        "idTag": "aircraft",
        "logo": "http://imgur.com/abQ9EaG.png",
        "title": "Aircraft Commerce",
      },
      Object {
        "_id": "4nNKyP6h6pMMjAyTMK",
        "idTag": "icc",
        "logo": "http://imgur.com/UmS4Irj.png",
        "title": "ICC Events",
      },
      Object {
        "_id": "5nNKyP6h6pMMjAyTMK",
        "idTag": "smartup",
        "logo": "http://imgur.com/BC6B8Bc.png",
        "title": "Smartup Visuals",
      },
      Object {
        "_id": "6nNKyP6h6pMMjAyTMK",
        "idTag": "aci",
        "logo": "http://imgur.com/N2LoYmv.png",
        "title": "ACI Events",
      },
      Object {
        "_id": "7nNKyP6h6pMMjAyTMK",
        "idTag": "aircraft",
        "logo": "http://imgur.com/abQ9EaG.png",
        "title": "Aircraft Commerce",
      },
      Object {
        "_id": "8nNKyP6h6pMMjAyTMK",
        "idTag": "icc",
        "logo": "http://imgur.com/UmS4Irj.png",
        "title": "ICC Events",
      },
      Object {
        "_id": "9nNKyP6h6pMMjAyTMK",
        "idTag": "smartup",
        "logo": "http://imgur.com/BC6B8Bc.png",
        "title": "Smartup Visuals",
      },
      Object {
        "_id": "10nNKyP6h6pMMjAyTMK",
        "idTag": "aci",
        "logo": "http://imgur.com/N2LoYmv.png",
        "title": "ACI Events",
      },
      Object {
        "_id": "11nNKyP6h6pMMjAyTMK",
        "idTag": "aircraft",
        "logo": "http://imgur.com/abQ9EaG.png",
        "title": "Aircraft Commerce",
      },
      Object {
        "_id": "12nNKyP6h6pMMjAyTMK",
        "idTag": "icc",
        "logo": "http://imgur.com/UmS4Irj.png",
        "title": "ICC Events",
      },
    ]
  }
  onUpdate={[Function]}
  value="3nNKyP6h6pMMjAyTMK">
  <Styled(Scroller)>
    <Scroller
      className="hHsXqK">
      <div
        className="hHsXqK"
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "WebkitOverflowScrolling": "touch",
            "backgroundColor": "transparent",
            "bottom": "0",
            "left": "0",
            "overflowX": "hidden",
            "overflowY": "auto",
            "paddingBottom": "40px",
            "position": "absolute",
            "right": "0",
            "top": "0",
          }
        }>
        <div
          className="display:flex pos:relative padding-l:7">
          <MenuItem
            image="http://imgur.com/BC6B8Bc.png"
            onClick={[Function]}
            pictureBgColor="white"
            pictureHeight={35}
            pictureWidth={35}
            title="Smartup Visuals"
            value="1nNKyP6h6pMMjAyTMK">
            <div>
              <div
                className="display:flex flex:items-center color:white padding:5 select:none cursor:pointer"
                onClick={[Function]}>
                <Thumb
                  color="white"
                  height={35}
                  icon="cactus"
                  image="http://imgur.com/BC6B8Bc.png"
                  width={35}>
                  <div
                    className="display:flex flex:items-center flex:content-center border:rounded color:white"
                    style={
                      Object {
                        "WebkitFlex": "0 0 35px",
                        "flex": "0 0 35px",
                        "height": "35px",
                        "width": "35px",
                      }
                    }>
                    <img
                      className="w:100p border:rounded"
                      src="http://imgur.com/BC6B8Bc.png" />
                  </div>
                </Thumb>
                <div
                  className="display:flex flex:column flex:content-center margin-l:5 min-w:0 line-h:130p font:roboto-condensed">
                  <div
                    className="select:text size:16 line-h:120p text:truncate text-color:backdrop-alt">
                    Smartup Visuals
                  </div>
                </div>
              </div>
            </div>
          </MenuItem>
        </div>
        <div
          className="display:flex pos:relative padding-l:7">
          <MenuItem
            image="http://imgur.com/N2LoYmv.png"
            onClick={[Function]}
            pictureBgColor="white"
            pictureHeight={35}
            pictureWidth={35}
            title="ACI Events"
            value="2nNKyP6h6pMMjAyTMK">
            <div>
              <div
                className="display:flex flex:items-center color:white padding:5 select:none cursor:pointer"
                onClick={[Function]}>
                <Thumb
                  color="white"
                  height={35}
                  icon="cactus"
                  image="http://imgur.com/N2LoYmv.png"
                  width={35}>
                  <div
                    className="display:flex flex:items-center flex:content-center border:rounded color:white"
                    style={
                      Object {
                        "WebkitFlex": "0 0 35px",
                        "flex": "0 0 35px",
                        "height": "35px",
                        "width": "35px",
                      }
                    }>
                    <img
                      className="w:100p border:rounded"
                      src="http://imgur.com/N2LoYmv.png" />
                  </div>
                </Thumb>
                <div
                  className="display:flex flex:column flex:content-center margin-l:5 min-w:0 line-h:130p font:roboto-condensed">
                  <div
                    className="select:text size:16 line-h:120p text:truncate text-color:backdrop-alt">
                    ACI Events
                  </div>
                </div>
              </div>
            </div>
          </MenuItem>
        </div>
        <div
          className="display:flex pos:relative padding-l:7">
          <SelectIndicator
            className="color:neutral-dim">
            <styled.div
              className="color:neutral-dim">
              <div
                className="color:neutral-dim iscjpC" />
            </styled.div>
          </SelectIndicator>
          <MenuItem
            image="http://imgur.com/abQ9EaG.png"
            onClick={[Function]}
            pictureBgColor="white"
            pictureHeight={35}
            pictureWidth={35}
            title="Aircraft Commerce"
            value="3nNKyP6h6pMMjAyTMK">
            <div>
              <div
                className="display:flex flex:items-center color:white padding:5 select:none cursor:pointer"
                onClick={[Function]}>
                <Thumb
                  color="white"
                  height={35}
                  icon="cactus"
                  image="http://imgur.com/abQ9EaG.png"
                  width={35}>
                  <div
                    className="display:flex flex:items-center flex:content-center border:rounded color:white"
                    style={
                      Object {
                        "WebkitFlex": "0 0 35px",
                        "flex": "0 0 35px",
                        "height": "35px",
                        "width": "35px",
                      }
                    }>
                    <img
                      className="w:100p border:rounded"
                      src="http://imgur.com/abQ9EaG.png" />
                  </div>
                </Thumb>
                <div
                  className="display:flex flex:column flex:content-center margin-l:5 min-w:0 line-h:130p font:roboto-condensed">
                  <div
                    className="select:text size:16 line-h:120p text:truncate text-color:backdrop-alt">
                    Aircraft Commerce
                  </div>
                </div>
              </div>
            </div>
          </MenuItem>
        </div>
        <div
          className="display:flex pos:relative padding-l:7">
          <MenuItem
            image="http://imgur.com/UmS4Irj.png"
            onClick={[Function]}
            pictureBgColor="white"
            pictureHeight={35}
            pictureWidth={35}
            title="ICC Events"
            value="4nNKyP6h6pMMjAyTMK">
            <div>
              <div
                className="display:flex flex:items-center color:white padding:5 select:none cursor:pointer"
                onClick={[Function]}>
                <Thumb
                  color="white"
                  height={35}
                  icon="cactus"
                  image="http://imgur.com/UmS4Irj.png"
                  width={35}>
                  <div
                    className="display:flex flex:items-center flex:content-center border:rounded color:white"
                    style={
                      Object {
                        "WebkitFlex": "0 0 35px",
                        "flex": "0 0 35px",
                        "height": "35px",
                        "width": "35px",
                      }
                    }>
                    <img
                      className="w:100p border:rounded"
                      src="http://imgur.com/UmS4Irj.png" />
                  </div>
                </Thumb>
                <div
                  className="display:flex flex:column flex:content-center margin-l:5 min-w:0 line-h:130p font:roboto-condensed">
                  <div
                    className="select:text size:16 line-h:120p text:truncate text-color:backdrop-alt">
                    ICC Events
                  </div>
                </div>
              </div>
            </div>
          </MenuItem>
        </div>
        <div
          className="display:flex pos:relative padding-l:7">
          <MenuItem
            image="http://imgur.com/BC6B8Bc.png"
            onClick={[Function]}
            pictureBgColor="white"
            pictureHeight={35}
            pictureWidth={35}
            title="Smartup Visuals"
            value="5nNKyP6h6pMMjAyTMK">
            <div>
              <div
                className="display:flex flex:items-center color:white padding:5 select:none cursor:pointer"
                onClick={[Function]}>
                <Thumb
                  color="white"
                  height={35}
                  icon="cactus"
                  image="http://imgur.com/BC6B8Bc.png"
                  width={35}>
                  <div
                    className="display:flex flex:items-center flex:content-center border:rounded color:white"
                    style={
                      Object {
                        "WebkitFlex": "0 0 35px",
                        "flex": "0 0 35px",
                        "height": "35px",
                        "width": "35px",
                      }
                    }>
                    <img
                      className="w:100p border:rounded"
                      src="http://imgur.com/BC6B8Bc.png" />
                  </div>
                </Thumb>
                <div
                  className="display:flex flex:column flex:content-center margin-l:5 min-w:0 line-h:130p font:roboto-condensed">
                  <div
                    className="select:text size:16 line-h:120p text:truncate text-color:backdrop-alt">
                    Smartup Visuals
                  </div>
                </div>
              </div>
            </div>
          </MenuItem>
        </div>
        <div
          className="display:flex pos:relative padding-l:7">
          <MenuItem
            image="http://imgur.com/N2LoYmv.png"
            onClick={[Function]}
            pictureBgColor="white"
            pictureHeight={35}
            pictureWidth={35}
            title="ACI Events"
            value="6nNKyP6h6pMMjAyTMK">
            <div>
              <div
                className="display:flex flex:items-center color:white padding:5 select:none cursor:pointer"
                onClick={[Function]}>
                <Thumb
                  color="white"
                  height={35}
                  icon="cactus"
                  image="http://imgur.com/N2LoYmv.png"
                  width={35}>
                  <div
                    className="display:flex flex:items-center flex:content-center border:rounded color:white"
                    style={
                      Object {
                        "WebkitFlex": "0 0 35px",
                        "flex": "0 0 35px",
                        "height": "35px",
                        "width": "35px",
                      }
                    }>
                    <img
                      className="w:100p border:rounded"
                      src="http://imgur.com/N2LoYmv.png" />
                  </div>
                </Thumb>
                <div
                  className="display:flex flex:column flex:content-center margin-l:5 min-w:0 line-h:130p font:roboto-condensed">
                  <div
                    className="select:text size:16 line-h:120p text:truncate text-color:backdrop-alt">
                    ACI Events
                  </div>
                </div>
              </div>
            </div>
          </MenuItem>
        </div>
        <div
          className="display:flex pos:relative padding-l:7">
          <MenuItem
            image="http://imgur.com/abQ9EaG.png"
            onClick={[Function]}
            pictureBgColor="white"
            pictureHeight={35}
            pictureWidth={35}
            title="Aircraft Commerce"
            value="7nNKyP6h6pMMjAyTMK">
            <div>
              <div
                className="display:flex flex:items-center color:white padding:5 select:none cursor:pointer"
                onClick={[Function]}>
                <Thumb
                  color="white"
                  height={35}
                  icon="cactus"
                  image="http://imgur.com/abQ9EaG.png"
                  width={35}>
                  <div
                    className="display:flex flex:items-center flex:content-center border:rounded color:white"
                    style={
                      Object {
                        "WebkitFlex": "0 0 35px",
                        "flex": "0 0 35px",
                        "height": "35px",
                        "width": "35px",
                      }
                    }>
                    <img
                      className="w:100p border:rounded"
                      src="http://imgur.com/abQ9EaG.png" />
                  </div>
                </Thumb>
                <div
                  className="display:flex flex:column flex:content-center margin-l:5 min-w:0 line-h:130p font:roboto-condensed">
                  <div
                    className="select:text size:16 line-h:120p text:truncate text-color:backdrop-alt">
                    Aircraft Commerce
                  </div>
                </div>
              </div>
            </div>
          </MenuItem>
        </div>
        <div
          className="display:flex pos:relative padding-l:7">
          <MenuItem
            image="http://imgur.com/UmS4Irj.png"
            onClick={[Function]}
            pictureBgColor="white"
            pictureHeight={35}
            pictureWidth={35}
            title="ICC Events"
            value="8nNKyP6h6pMMjAyTMK">
            <div>
              <div
                className="display:flex flex:items-center color:white padding:5 select:none cursor:pointer"
                onClick={[Function]}>
                <Thumb
                  color="white"
                  height={35}
                  icon="cactus"
                  image="http://imgur.com/UmS4Irj.png"
                  width={35}>
                  <div
                    className="display:flex flex:items-center flex:content-center border:rounded color:white"
                    style={
                      Object {
                        "WebkitFlex": "0 0 35px",
                        "flex": "0 0 35px",
                        "height": "35px",
                        "width": "35px",
                      }
                    }>
                    <img
                      className="w:100p border:rounded"
                      src="http://imgur.com/UmS4Irj.png" />
                  </div>
                </Thumb>
                <div
                  className="display:flex flex:column flex:content-center margin-l:5 min-w:0 line-h:130p font:roboto-condensed">
                  <div
                    className="select:text size:16 line-h:120p text:truncate text-color:backdrop-alt">
                    ICC Events
                  </div>
                </div>
              </div>
            </div>
          </MenuItem>
        </div>
        <div
          className="display:flex pos:relative padding-l:7">
          <MenuItem
            image="http://imgur.com/BC6B8Bc.png"
            onClick={[Function]}
            pictureBgColor="white"
            pictureHeight={35}
            pictureWidth={35}
            title="Smartup Visuals"
            value="9nNKyP6h6pMMjAyTMK">
            <div>
              <div
                className="display:flex flex:items-center color:white padding:5 select:none cursor:pointer"
                onClick={[Function]}>
                <Thumb
                  color="white"
                  height={35}
                  icon="cactus"
                  image="http://imgur.com/BC6B8Bc.png"
                  width={35}>
                  <div
                    className="display:flex flex:items-center flex:content-center border:rounded color:white"
                    style={
                      Object {
                        "WebkitFlex": "0 0 35px",
                        "flex": "0 0 35px",
                        "height": "35px",
                        "width": "35px",
                      }
                    }>
                    <img
                      className="w:100p border:rounded"
                      src="http://imgur.com/BC6B8Bc.png" />
                  </div>
                </Thumb>
                <div
                  className="display:flex flex:column flex:content-center margin-l:5 min-w:0 line-h:130p font:roboto-condensed">
                  <div
                    className="select:text size:16 line-h:120p text:truncate text-color:backdrop-alt">
                    Smartup Visuals
                  </div>
                </div>
              </div>
            </div>
          </MenuItem>
        </div>
        <div
          className="display:flex pos:relative padding-l:7">
          <MenuItem
            image="http://imgur.com/N2LoYmv.png"
            onClick={[Function]}
            pictureBgColor="white"
            pictureHeight={35}
            pictureWidth={35}
            title="ACI Events"
            value="10nNKyP6h6pMMjAyTMK">
            <div>
              <div
                className="display:flex flex:items-center color:white padding:5 select:none cursor:pointer"
                onClick={[Function]}>
                <Thumb
                  color="white"
                  height={35}
                  icon="cactus"
                  image="http://imgur.com/N2LoYmv.png"
                  width={35}>
                  <div
                    className="display:flex flex:items-center flex:content-center border:rounded color:white"
                    style={
                      Object {
                        "WebkitFlex": "0 0 35px",
                        "flex": "0 0 35px",
                        "height": "35px",
                        "width": "35px",
                      }
                    }>
                    <img
                      className="w:100p border:rounded"
                      src="http://imgur.com/N2LoYmv.png" />
                  </div>
                </Thumb>
                <div
                  className="display:flex flex:column flex:content-center margin-l:5 min-w:0 line-h:130p font:roboto-condensed">
                  <div
                    className="select:text size:16 line-h:120p text:truncate text-color:backdrop-alt">
                    ACI Events
                  </div>
                </div>
              </div>
            </div>
          </MenuItem>
        </div>
        <div
          className="display:flex pos:relative padding-l:7">
          <MenuItem
            image="http://imgur.com/abQ9EaG.png"
            onClick={[Function]}
            pictureBgColor="white"
            pictureHeight={35}
            pictureWidth={35}
            title="Aircraft Commerce"
            value="11nNKyP6h6pMMjAyTMK">
            <div>
              <div
                className="display:flex flex:items-center color:white padding:5 select:none cursor:pointer"
                onClick={[Function]}>
                <Thumb
                  color="white"
                  height={35}
                  icon="cactus"
                  image="http://imgur.com/abQ9EaG.png"
                  width={35}>
                  <div
                    className="display:flex flex:items-center flex:content-center border:rounded color:white"
                    style={
                      Object {
                        "WebkitFlex": "0 0 35px",
                        "flex": "0 0 35px",
                        "height": "35px",
                        "width": "35px",
                      }
                    }>
                    <img
                      className="w:100p border:rounded"
                      src="http://imgur.com/abQ9EaG.png" />
                  </div>
                </Thumb>
                <div
                  className="display:flex flex:column flex:content-center margin-l:5 min-w:0 line-h:130p font:roboto-condensed">
                  <div
                    className="select:text size:16 line-h:120p text:truncate text-color:backdrop-alt">
                    Aircraft Commerce
                  </div>
                </div>
              </div>
            </div>
          </MenuItem>
        </div>
        <div
          className="display:flex pos:relative padding-l:7">
          <MenuItem
            image="http://imgur.com/UmS4Irj.png"
            onClick={[Function]}
            pictureBgColor="white"
            pictureHeight={35}
            pictureWidth={35}
            title="ICC Events"
            value="12nNKyP6h6pMMjAyTMK">
            <div>
              <div
                className="display:flex flex:items-center color:white padding:5 select:none cursor:pointer"
                onClick={[Function]}>
                <Thumb
                  color="white"
                  height={35}
                  icon="cactus"
                  image="http://imgur.com/UmS4Irj.png"
                  width={35}>
                  <div
                    className="display:flex flex:items-center flex:content-center border:rounded color:white"
                    style={
                      Object {
                        "WebkitFlex": "0 0 35px",
                        "flex": "0 0 35px",
                        "height": "35px",
                        "width": "35px",
                      }
                    }>
                    <img
                      className="w:100p border:rounded"
                      src="http://imgur.com/UmS4Irj.png" />
                  </div>
                </Thumb>
                <div
                  className="display:flex flex:column flex:content-center margin-l:5 min-w:0 line-h:130p font:roboto-condensed">
                  <div
                    className="select:text size:16 line-h:120p text:truncate text-color:backdrop-alt">
                    ICC Events
                  </div>
                </div>
              </div>
            </div>
          </MenuItem>
        </div>
      </div>
    </Scroller>
  </Styled(Scroller)>
</IconBar>
`;
